src/frontend/context/Valutakurs/ValutakursSkjemaContext.tsx

```diff
@@ -28,23 +29,29 @@ const erValutakursDatoGyldig = (
     felt: FeltState<string | undefined>
 ): FeltState<string | undefined> =>
     !isEmpty(felt.verdi) ? ok(felt) : feil(felt, 'Valutakursdato er påkrevd, men mangler input');
-const erValutakursGyldig = (felt: FeltState<string | undefined>): FeltState<string | undefined> => {
-    if (!felt.verdi || isEmpty(felt.verdi) || typeof felt.verdi != 'string') {
-        return feil(felt, 'Valutakurs er påkrevd, men mangler input');
-    }
-    const nyKurs = konverterSkjemaverdiTilDesimal(felt.verdi);
-    if (!nyKurs) {
-        return feil(felt, 'Valutakurs er påkrevd, men mangler input');
-    }
-    if (!isNumeric(nyKurs)) {
-        return feil(felt, `Valutakurs innholder ugyldige verdier, kurs: ${felt.verdi}`);
-    }
-    if (tellAntallDesimaler(nyKurs) !== 4) {
-        return feil(felt, `Valutakurs må ha 4 desimaler, kurs: ${felt.verdi}`);
-    }
-    const kurs = Number(nyKurs);
-    if (kurs < 0) {
-        return feil(felt, `Kan ikke registrere negativt kurs: ${felt.verdi}`);
+const erValutakursGyldig = (
+    felt: FeltState<string | undefined>,
+    skalValideres: boolean
+): FeltState<string | undefined> => {
+    if (skalValideres) {
+        if (!felt.verdi || isEmpty(felt.verdi) || typeof felt.verdi != 'string') {
+            return feil(felt, 'Valutakurs er påkrevd, men mangler input');
+        }
+        const nyKurs = konverterSkjemaverdiTilDesimal(felt.verdi);
+        if (!nyKurs) {
+            return feil(felt, 'Valutakurs er påkrevd, men mangler input');
+        }
+        if (!isNumeric(nyKurs)) {
+            return feil(felt, `Valutakurs innholder ugyldige verdier, kurs: ${felt.verdi}`);
+        }
+        if (tellAntallDesimaler(nyKurs) === 0) {
```

Comment:
Går vi bort ifra kravet om at valutakurs ikke trenger å ha 4 desimaler?
Ref: https://favro.com/organization/98c34fb974ce445eac854de0/1844bbac3b6605eacc8f5543?onShow=mycards&card=Tea-9071

Reply:
Ja, jeg tror det! Dobbeltsjekker med Gunn 👍

Reply:
De eneste valutakursene som nå manuelt skal fylles inn er islandske kroner (ISK) og disse kopieres over fra et Excel-ark saksbehandlerne har. Valutakursen som står oppført der har ofte mer enn 4 desimaler. Derfor kan det bli litt rart om de må redusere antall desimaler manuelt etter å ha kopiert over. Da slipper de å tenke på avrunding osv. 

Burde kanskje vært minimum 4 da og ikke nøyaktig 4 så jeg hører med Gunn for å få det avklart!